templates/repo/branch/list.tmpl

```diff
@@ -193,4 +225,27 @@
 		</form>
 	</div>
 </div>
+
+<div class="ui g-modal-confim modal" id="rename-branch-modal">
+	<div class="header">
+		{{.locale.Tr "repo.settings.rename_branch"}}
+	</div>
+	<div class="content">
+		<form class="ui form" id="rename-branch-form" action="{{$.Repository.Link}}/settings/rename_branch" method="post">
+			{{.CsrfTokenHtml}}
+			<div class="field">
+				<label>
+					<span class="text" id="modal-rename-branch-to-span" data-rename-branch-to="{{.locale.Tr "repo.branch.rename_branch_to"}}"></span>
+				</label>
+			</div>
+			<input id="from" name="from" value="" type="hidden" required>
+			<div class="required field">
+				<input id="to" name="to" required>
```

Comment:
The screenshot shows that it is very wide?

Reply:
![image](https://user-images.githubusercontent.com/33891828/235277027-f3ef0665-1c20-4e77-b7c8-9d45404ff50e.png)
seems the width of `ui.mini.modal` is enough?  [8d7c7da](https://github.com/go-gitea/gitea/pull/24380/commits/8d7c7dabd636319864af7772f1a186328b20cc09)